client/my-sites/sidebar/sidebar.jsx

```diff
@@ -146,11 +146,21 @@ export class MySitesSidebar extends Component {
 			return null;
 		}
 
+		let sparkline;
+		if ( ! site.jetpack && ! this.isItemLinkSelected( 'stats' ) ) {
```

Comment:
Looks like the `masterbar` param might not have made its way into JP:

<img width="250" alt="admin_php__224x48__and_my_stats_ _wordpress_com" src="https://cloud.githubusercontent.com/assets/22080/21598232/c9e20230-d10d-11e6-9360-e747cbe68738.png">